### PR TITLE
The old breeze video and related assets are removed

### PR DESCRIPTION
**Pull request by potiuk**
The video and related assets are not valid in most cases any more.
We discussed re-recording Breeze videe after we complete Python
conversion.

This PR was extracted out of #25678 as it outgrew it's initial scope

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @rossturk

**Comment by potiuk** [comment]
Removing a lot :)

**Comment by potiuk** [comment]
Getting close to remove all old-breeze stuff :)

File: BREEZE.rst [hunk 1]
```diff
@@ -806,32 +718,6 @@ through ``breeze testing docker-compose-tests`` command.
...
-Breeze helps to setup KinD cluster for testing, setting up virtualenv and downloads the right tools
-automatically to run the tests.
-
-You can:
-
-* Manage KinD Kubernetes cluster and deploy Airflow to KinD cluster ``./breeze-legacy kind-cluster`` commands
-* Run Kubernetes tests  specified with ``./breeze-legacy kind-cluster tests`` command
-* Enter the interactive kubernetes test environment with ``./breeze-legacy kind-cluster shell`` command
-
-This is described in detail in `Testing Kubernetes <TESTING.rst#running-tests-with-kubernetes>`_.
```
Comment (jedcunningham) [inline BREEZE.rst:822]: Did you mean to remove this section?

Comment (potiuk) [reply BREEZE.rst:822]: Yep. This is very temporary and it is going to be replaced by #25678

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**